### ISSUE TITLE
Update Buttons and Links for Accessibility

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7754]
# Project Status Overview

### Project Objective:

We completed an audit of the buttons and links found in the mobile app in November of 2023. Those results can be found [in this Google sheet](https://docs.google.com/spreadsheets/d/1uOF4dn3yptsp7VnxwG7owy2p0GDHjmBq1oBR8MIVvWg/edit?usp=sharing). As a result of those findings and our internal meeting on January 18, it was determined that an epic should be created so that we could break the necessary updates down by screens / areas within the app.

**Comment by timwright12** [comment]
closing as not actionable
